inference_rules.adoc

```diff
@@ -440,12 +440,16 @@ suite result, e.g. “a MLPerf result of 4.5.”
 
 The Network division inherits all requirements from the Closed division and imposes further constraints.
 In the Network division the SUT is connected to the Loadgen system over a network fabric. The Query Dispatch Library (QDL) component is a submitter-implemented SUT proxy that runs on the Loadgen system. The Network division supports only the Datacenter suite.
+Non-conforming network submission should be submitted to Open category, under the Open category constraints.
 
 ==== QDL Constraints
 
-* The QDL is not allowed to do any pre-processing.
+* The QDL is not allowed to do any pre-processing. e.g. changing of precision, or data layout.
```

Comment:
add batching here?

Reply:
let's be specific on batching as it is now since it is a point under dispute